app/models/computacenter/user_change_generator.rb

```diff
@@ -10,6 +10,7 @@ def generate!
       change = Computacenter::UserChange.new(consolidated_attributes)
       change.add_original_fields_from(last_change_for_user) if last_change_for_user.present?
       change.save!
+      NotifyComputacenterOfLatestChangeForUserJob.perform_later(@user.id) if Settings.computacenter.service_now_user_import_api.endpoint.present?
```

Comment:
Personal taste but I find this a bit jarring in the main logic flow and think it would be better as a separate private method e.g.`notify_computacenter` or the `Settings...` bit made into a private method e.g. `service_now_api_configured?`
I realise that moving it into the UserChange model as `save_and_notify_computacenter!` would feel wrong as well given your queueing against the `user_id` for good reason. Not a biggie, it just stopped me in my tracks where the rest of the changes flowed great.

Reply:
Totally agree - done in f8fb0650